bootstrap.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+# The -e option would make our script exit with an error if any command
+# fails while the -x option makes verbosely it output what it does
+
+# Install Pipenv, the -n option makes sudo fail instead of asking for a
+# password if we don’t have sufficient privileges to run it
+sudo -n dnf install -y pipenv
+
+# run our app. Nohup and "&" are used to let the setup script finish
+# while our app stays up. The app logs will be collected in nohup.out
+ nohup pipenv run python manage.py runserver 0.0.0.0:8000 &
```

Comment:
Given that you don't have a Django application, I don't see how this line would work at this stage.

Reply:
I tried to run "vagrant up " in my computer and it seem to be working with this line . 
Should I delete it ?

Reply:
Its probably showing you an error message because of this line, or perhaps dumping it into `nohup.out`.

I think you should add this line until you have an actual reason to do it